### ISSUE TITLE
Shareable link still needed?

### ISSUE THREAD

**Issue by pipeeeeees** [austinyen56/Waze-F1-VO#1]
I can turn mp3s into a shareable link if still desired.

**Comment by austinyen56** [comment]
Yea sure! That will be nice :)

**Comment by pipeeeeees** [comment]
Missing `1000meters.mp3`, `1500meters.mp3`, missing a few 'Start Drives', so I just recycled the first four over the total 10. `TinkerPoints.mp3` is not a valid mp3 file, `TickerPoints.mp3` is - I scrapped it as it was too large a file size and Waze traditionally defaults this to its usual "rerouting" file anyway. Had to convert the pack to 46k bitrate to maximize quality on the key voice lines and lower bitrate on some of the less important voices to fit the mp3 files into a size <0.8MB altogether (the max Waze servers appear to currently allow for custom voice packs). The following link is the highest quality I could get onto the server: https://waze.com/ul?acvp=561c4d75-666b-4f95-b5fa-de7b85646e01
